verilog/analysis/checkers/token_stream_lint_rule.cc

```diff
@@ -69,7 +70,16 @@ void TokenStreamLintRule::HandleSymbol(const verible::Symbol& symbol,
   verible::matcher::BoundSymbolManager manager;
   if (StringLiteralMatcher().Matches(symbol, &manager)) {
```

Comment:
To keep the indentation level flat, we can formulate this first test with an early return
```
  if (!StringLiteralMatcher().Matches(symbol, &manager))
    return;
```